asset/sass/_elements.scss

```diff
@@ -30,3 +30,18 @@ pre[class*="language-"] {
     margin-top: $padding-large-vertical * 2;
     margin-bottom: $padding-large-vertical * 2;
 }
+
+.dropdown_packagename {
+  font-size: .8em;
+  color: #666666;
```

Comment:
I think, the color value can be replaced with the Sass variable `$text-color-secondary`, which is `#777`.

Reply:
Done.